docs/pages/patterns/combobox-input.md

```diff
@@ -34,22 +34,57 @@ If the entries are not validated by the application, users can also enter custom
-            <li role="option" tabindex="0" class="fd-list__item is-selected">
-                <span class="fd-list__title">
-                    <span class="fd-list__bold">App</span>le
-                </span>
-            </li>
-            <li role="option" tabindex="0" class="fd-list__item">
-                <span class="fd-list__title">Orange</span>
-            </li>
-            <li role="option" tabindex="0" class="fd-list__item">
-                <span class="fd-list__title">Banana</span>
-            </li>
-            <li role="option" tabindex="0" class="fd-list__item">
-                <span class="fd-list__title">Kiwi</span>
-            </li>
-        </ul>
+    <div class="fd-popover__wrapper">
+      <ul class="fd-list fd-list--dropdown" role="listbox">
+          <li role="option" tabindex="0" class="fd-list__item is-selected">
+              <span class="fd-list__title">
+                  <span class="fd-list__bold">App</span>le
+              </span>
+          </li>
+          <li role="option" tabindex="0" class="fd-list__item">
+              <span class="fd-list__title">Orange</span>
+          </li>
+          <li role="option" tabindex="0" class="fd-list__item">
+              <span class="fd-list__title">Banana</span>
+          </li>
+          <li role="option" tabindex="0" class="fd-list__item">
+              <span class="fd-list__title">Kiwi</span>
+          </li>
+          <li role="option" tabindex="0" class="fd-list__item">
+              <span class="fd-list__title">Kiwi</span>
+          </li>
+          <li role="option" tabindex="0" class="fd-list__item">
+              <span class="fd-list__title">Kiwi</span>
+          </li>
+          <li role="option" tabindex="0" class="fd-list__item">
+              <span class="fd-list__title">Kiwi</span>
+          </li>
+          <li role="option" tabindex="0" class="fd-list__item">
+              <span class="fd-list__title">Kiwi</span>
+          </li>
+          <li role="option" tabindex="0" class="fd-list__item">
+              <span class="fd-list__title">Kiwi</span>
+          </li>
+          <li role="option" tabindex="0" class="fd-list__item">
+              <span class="fd-list__title">Kiwi</span>
+          </li>
+          <li role="option" tabindex="0" class="fd-list__item">
+              <span class="fd-list__title">Kiwi</span>
+          </li>
+          <li role="option" tabindex="0" class="fd-list__item">
+              <span class="fd-list__title">Kiwi</span>
+          </li>
+          <li role="option" tabindex="0" class="fd-list__item">
+              <span class="fd-list__title">Kiwi</span>
+          </li>
+          <li role="option" tabindex="0" class="fd-list__item">
+              <span class="fd-list__title">Kiwi</span>
+          </li>
+          <li role="option" tabindex="0" class="fd-list__item">
+              <span class="fd-list__title">Kiwi</span>
+          </li>
+      </ul>
+    </div>
```

Comment:
Could you use different fruits? And if you want to show the scroll case you prob want to add max-height

Reply:
fixed